examples/00-fluent/modeling_cavitation.py

```diff
@@ -104,56 +103,48 @@
 
 solver.setup.materials.fluid["water"] = water
 
-# Copy water vapor from the materials database and modify the properties of your
-# local copy.
-
 solver.setup.materials.database.copy_by_name(type="fluid", name="water-vapor")
 
-# Set 0.02558 kg/m3 for Density and 1.26e-06 kg/m–s for Viscosity.
-
 water_vapor = {"density": {"value": 0.02558}, "viscosity": {"value": 1.26e-06}}
 
 solver.setup.materials.fluid["water-vapor"] = water_vapor
 
 ###############################################################################
 # Phases
 # ~~~~~~
-# Change phase names to "liquid" and "water-vapor".
+# Change phase names to "liquid" for the primary phase and "water-vapor" for the
+# secondary phase. Enable the cavitation model and set the number of mass
+# transfer mechanisms to 1, liquid as the from phase, vapor as the to phase and
+# cavitation as the mechanism.
 
 solver.tui.define.phases.set_domain_properties.change_phases_names("vapor", "liquid")
 
-# Specify liquid water as the primary phase.
-
 solver.tui.define.phases.set_domain_properties.phase_domains.liquid.material(
     "yes", "water"
 )
 
-# Specify water vapor as the secondary phase.
-
 solver.tui.define.phases.set_domain_properties.phase_domains.vapor.material(
     "yes", "water-vapor"
 )
 
-# Enable the cavitation model
-
-# Set the number of mass transfer mechanisms to 1, set liquid as the from phase,
-# set vapor as the to phase and cavitation as the mechanism.
-
 solver.tui.define.phases.set_domain_properties.interaction_domain.heat_mass_reactions.mass_transfer(
     1, "liquid", "vapor", "cavitation", "1", "no", "no", "no"
 )
 
 ###############################################################################
 # Boundary Conditions
 # ~~~~~~~~~~~~~~~~~~~
-# Set momentum and turbulence boundary conditions for the first inlet
 
```

Comment:
This extra empty line is causing the text to become a code block:
![image](https://github.com/ansys/pyfluent/assets/132297401/8ff3afc8-5675-46cc-b5a9-a8d9b9ad2a47)